example-change-logo/artifacthub-pkg.yml

```diff
@@ -0,0 +1,12 @@
+version: 0.0.1
+name: headlamp_example_change_logo
+displayName: Change Logo Example
+createdAt: "2024-07-18T05:35:50Z"
+logoURL: "https://github.com/kubernetes/kompose/raw/main/docs/assets/images/logo.png"
+description: Allows to change the Headlamp logo. This is an plugin from the official examples.
+annotations:
+https://github.com/headlamp-k8s/plugins/releases/download/untagged-65b9c873f696670f9107/change-logo-0.0.1.tar.gz
```

Comment:
Not a valid YAML.

Reply:
Must have made a typo before adding/pushing...